pkg/oc/bootstrap/docker/up.go

```diff
@@ -135,6 +135,12 @@ var (
 		"template service broker apiserver": "install/templateservicebroker/previous/apiserver-template.yaml",
 	}
 
+	// consoleTemplateLocations are templates that will be registered in an internal namespace
+	// for the web console
+	consoleTemplateLocations = map[string]string{
+		"web console server template": "install/origin-web-console/console-template.yaml",
+	}
```

Comment:
any reason you didn't just put this in the "internalCurrentTemplateLocations" array?
(the comments about it only being used w/ the service catalog is requested are inaccurate, you can clean them up :) )

Reply:
Right now the template is only imported on 3.9+ clusters, which is why it's in a separate map. We could always import it, though.

Reply:
OK I see `internalCurrentTemplateLocations` only installs in 3.9. I'll update.